gprofiler/profilers/java.py

```diff
@@ -966,11 +1011,22 @@ def _profile_process(self, process: Process, duration: int, spawned: bool) -> Pr
             self._jattach_timeout,
             self._ap_mcache,
         ) as ap_proc:
-            stackcollapse = self._profile_ap_process(ap_proc, comm, duration)
+            stackcollapse, process_uptime_str = self._profile_ap_process(ap_proc, comm, duration)
+
+        if process_uptime_str is not None:
+            if appid:
+                appid += ", "
+            appid = str(appid) + process_uptime_str
```

Comment:
If `appid` was None then it's `str(None)` here, I guess in this case use `java: {process_uptime_str}`?